native/dorgtr.go

```diff
@@ -0,0 +1,92 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import "github.com/gonum/blas"
+
+// Dorgtr generates a real orthogonal matrix Q which is defines as the product
+// of n-1 elementary reflectors of order N as returned by Dsytrd.
+//
+// If uplo == blas.Upper
+//  Q = H[n-1] * ... * H[1] * H[0]
+// If uplo == blas.Lower
+//  L = H[0] * ... * H[n-1]
+//
```

Comment:
Could you delete this line and connect the ifs with where to read as a single sentence?